ddl/db_integration_test.go

```diff
@@ -547,7 +547,17 @@ func (s *testIntegrationSuite2) TestNullGeneratedColumn(c *C) {
 		") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin")
 
 	tk.MustExec("insert into t values()")
-	tk.MustExec("alter table t add index idx_c(c)")
+
+	// --------------------------------------------------------
```

Comment:
Why add this line?

Reply:
For the sake of readability, this line and line 559 form a code block.

Would it be better if I change into
```go
// Currently TiDB is not support adding index on virtual generated column.
// Tests below should be uncommented in the future.
// *old tests go here*
*new tests go here*
```

Reply:
For the sake of readability, this line and line 559 form a code block.

Would it be better if I change it into
```
// Currently TiDB not support adding an index on the virtual generated column.
// Tests below should be uncommented in the future.
// *old tests go here*
*new tests go here*
```